### PR TITLE
Preview is not available for any post type

### PR DESCRIPTION
**Pull request by roborourke**
Currently none of the new posts/post types can be previewed before publishing. Viewing a post only works after publishing it, preview opens a blank page.

### CODE REVIEW

**Comment by roborourke** [comment]
@joehoyle slightly nuts perhaps but works, could use some review on this!

File: functions.php [hunk 1]
```diff
@@ -4,8 +4,12 @@
 
 add_action( 'wp_enqueue_scripts', function() {
 	wp_enqueue_script( 'app', get_template_directory_uri() . '/dist/main.js', array(), wp_get_theme()->Version, true );
+	wp_localize_script( 'app', 'app_data', array(
+		'api_url' => home_url( '/wp-json' ),
```
Comment (rmccue) [inline functions.php]: Should be able to use `rest_url()` here instead.

File: js/api.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ module.exports = {
 			data: null
 		}
 
+		console.log(data)
```
Comment (joehoyle) [inline js/api.js]: Can this be removed please?

**Comment by joehoyle** [comment]
@sanchothefat one hickup then we are ready to cole, thanks!

**Comment by roborourke** [comment]
:+1:

**Pull request merged**